doc/webservices/upgrade.rst

```diff
@@ -479,6 +480,75 @@ should be empty if everything went fine.
 
 .. _upgrade-api-status-method:
 
+
+Asking to skip the tests 
+=========================
+
+This action asks the Upgrade Platform to skip the tests for your request.
+If you don't want Odoo to test and validate the migration, you can bypass the testing stage and directly get the migrated dump.
+
+The ``skip_test`` method
+----------------------
+
+.. py:function:: https://upgrade.odoo.com/database/v1/skip_test
+
+    Skip the tests, deliver the upgraded dump, and set the state to 'delivered'
+
+    :param str key: (required) your private key
+    :param str request: (required) your request id
+    :return: request result
+    :rtype: JSON dictionary
+
+The request id and the private key are obtained using the :ref:`create method
+<upgrade-api-create-method>`
+
+The result is a JSON dictionary containing the list of ``failures``, which
+should be empty if everything went fine.
+
+.. rst-class:: setup doc-aside
+
+.. switcher::
+
+    .. code-block:: python
+
+        from urllib import urlencode
+        from io import BytesIO
+        import pycurl
+        import json
```

Comment:
```python
from urllib import parse, request

URL = "https://upgrade.odoo.com/database/v1/skip_test"
data = parse.urlencode({'request': '10534', 'key': 'Aw7pItGVKFuZ_FOR3U8VFQ=='}).encode()
response = request.urlopen(URL, data).read()
...
```

Reply:
Thank you.
However, for the moment all the API calls are similar and uses pycurl and I prefer let them aligned and process step by step.

Next step is to update all requests, I created a task in the migration project to use request instead of pycurl. (see https://www.odoo.com/web#id=1942682&action=333&active_id=1205&model=project.task&view_type=form&menu_id=4720)

Note: please don't merge this branch for the moment as this PR should be validate with the PR in the migration project.

Thanks a lot for the feedback. 
Best regards

Reply:
@Elkasitu I just updated the PR.
However, I'am using request**s**  (already in the odoo requirements)

Reply:
Can you please update all code snippets to use `requests` instead of `pycurl`?

Reply:
Also, remove all python2 versions as code using `requests` should be compatible python2 and python3.